README.md

```diff
@@ -34,25 +36,32 @@ add in your other classes, and let the schedule generator create schedules for y
 
 Follow these steps to start a local Django server using a PostgreSQL database:
 
+0) For Windows, use powershell instead of command prompt.
+    - Some commands don't always work on command prompt.
 1) If you don’t have it already, download Python from [here](https://www.python.org/downloads/).
-2) To install the packages, you first need to make a virtual environment for Python, which will help us ensure that our libraries & Python versions are unified. You can do so be by running:
+2) Clone the current project: `git clone https://github.com/aggie-coding-club/Rev-Registration.git`
+    - This will clone the project into the current directory
+3) To install the packages, you first need to make a virtual environment for Python, which will help us ensure that our libraries & Python versions are unified. You can do so by running:
+    - It is recommended to put the virtual environment in the root directory: `./Rev-Registration`
```

Comment:
I'd move this as a bullet point under the `python3 -m venv env` or in the instructions for the step itself since this isn't actually a command you run, just a pointer

Reply:
Changed the wording/levels a bit